code/leslie/Python/lab02.py

```diff
@@ -1,13 +1,18 @@
 #Version1
 ones_digits = ["","one","two","three","four","five","six","seven","eight","nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"]
-tens_digits = ["", "ten", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety", "one hundred"]
-num = 67
+tens_digits = ["", "ten", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety", ""]
```

Comment:
Needs Work:

* Submission of `713` results in `seven hundred ten three`. Should be something like `seven hundred thirteen`.
```
PS C:\Users\Bruce\Programming\class_swordfish\code\leslie\Python> python .\lab02.py
Enter a number: 713
seven hundred ten three
PS C:\Users\Bruce\Programming\class_swordfish\code\leslie\Python>
```